package.json

```diff
@@ -6,6 +6,7 @@
     "@material-ui/icons": "^1.0.0-beta.43",
     "enzyme": "^3.3.0",
     "enzyme-adapter-react-16": "^1.1.1",
+    "material-design-icons": "^3.0.1",
```

Comment:
I tried to uninstall this library again and the icon is still there. Please uninstall it, then I merge